processing/src/main/java/io/druid/query/aggregation/cardinality/types/LongCardinalityAggregatorColumnSelectorStrategy.java

```diff
@@ -30,12 +31,16 @@
   @Override
```

Comment:
Same comment about javadoc about perf, as for `DoubleCardinalityAggregatorColumnSelectorStrategy`.

Reply:
done.